app/frontend/src/cljs/teet/meeting/meeting_view.cljs

```diff
@@ -150,6 +153,62 @@
                       :attach-to [:meeting-decision id]
                       :files files}]])
 
+(defn meeting-view-container
+  ([param]
+   [meeting-view-container param theme-colors/gray-light])
+  ([{:keys [text-color content open? heading heading-button children after-children-component]
+     :or {text-color :inherit
+          open? false}}
+    bg-color]
+   (r/with-let [open? (r/atom open?)
+                toggle-open! #(do
+                                (.stopPropagation %)
+                                (swap! open? not))]
+               [:div {:style {:border-bottom (str "1px solid " theme-colors/gray-lighter)
+                              :padding-bottom "1rem"
+                              :padding-top "1rem"}}
+                [:div {:class (<class common/hierarchical-heading-container2 bg-color text-color (and
+                                                                                                   (or (seq children) after-children-component)
+                                                                                                   @open?))}
+                 [:div
+                  [:div {:style {:width "100%"
+                                 :display :flex
+                                 :justify-content :space-between
+                                 :align-items :center}}
+                   [:div {:style {:flex-grow 1
+                                  :text-align :start}}
+                    heading]
+                   (when (and heading-button @open?)
+                     [:div {:style {:flex-grow 0}
+                            :on-click (fn [e]
+                                        (.stopPropagation e))}
+                      heading-button])
+                   [:div {:style {:margin-left "1rem"}}
+                    [buttons/button-primary
+                     {:size :small
+                      :on-click toggle-open!}
+                     [(if @open? icons/hardware-keyboard-arrow-up icons/hardware-keyboard-arrow-down)]]
+                    ]]]
+                 (when content
+                   [Collapse {:in @open?
+                              :mount-on-enter true}
+                    [:div {:style {:padding "1rem"}}
+                     content]])]
+
+                (when (or children after-children-component)
+                  [Collapse {:in @open?
+                             :mount-on-enter true}
+                   [:div {:class (<class common/hierarchical-child-container)
+                          :style {:border-top (str "1px solid " theme-colors/gray-lighter)}}
+                    (doall
+                      (for [child children]
+                        (with-meta
+                          (if (vector? child)                         ;;Check if it's component and render that instaed
+                            child
```

Comment:
Is all of this stuff necessary? At a quick glance I can't find a place where this component gets a component as a child. So some of these checks could be removed.